### ISSUE TITLE
Желаю чтобы команды установки/снятия/перехода закладок работали в поле запроса схемы компоновки

### ISSUE THREAD

**Issue by tormozit** [SeiOkami/OneS#219]
8.3.23
https://partners.v8.1c.ru/forum/t/2134105/m/2134105
Открываем конструктор любой схемы компоновки (например из нового отчета). Создаем набор данных типа "Запрос". Пишем там любой текст. Нажимаем сочетание клавиш ALT+F2. Происходит ничего.

Желаю, чтобы команды установки/снятия/перехода закладок работали в этом поле, т.к. оно часто используется для чтения текста запроса.

![изображение](https://github.com/SeiOkami/OneS/assets/11896251/c65ad4c5-960b-4b61-9135-3ddc5f5c3f5c)

**Comment by Morkhe** [comment]
На 8.3.22 работает, но сама закладка появляется только после дергания разделителя формы (см. скрин).
(Что там на партнерке - посмотреть не могу).
![image](https://github.com/SeiOkami/OneS/assets/24790188/962850c9-4664-400b-a9d8-cd3199dcb4c7)

**Comment by tormozit** [comment]
@Morkhe Да. Я позабыл про этот баг платформы. Регистрировал его тут https://www.hostedredmine.com/issues/958568
